node/src/Transaction.ts

```diff
@@ -736,6 +737,21 @@ export class BaseTransaction<T extends BaseTransaction<T>> {
         return this.addAndReturn(createSInter(keys), true);
     }
 
+    /**
+     * Gets the cardinality of the intersection of all the given sets.
+     *
+     * See https://valkey.io/commands/sintercard/ for more details.
+     *
+     * @param keys - The keys of the sets.
+     *
+     * Command Response - The cardinality of the intersection result. If one or more sets do not exist, `0` is returned.
+     *
+     * since Valkey version 7.0.0.
```

Comment:
Do we put `since` before params (like in java) or after return (like in python)?

Reply:
the existing docs put it between the returns and the example which is what I've done here. I'm impartial as to where we put it but this seems like a good place to me

Reply:
Put it anywhere you find suitable and we will update it for all commands in node client later.